x-pack/solutions/security/plugins/security_solution/public/overview/components/detection_response/utils.tsx

```diff
@@ -5,13 +5,24 @@
  * 2.0.
  */
 
+import type { EuiThemeComputed } from '@elastic/eui';
+
+// @deprecated, use getSeverityColor instead
 export const SEVERITY_COLOR = {
   critical: '#E7664C',
   high: '#DA8B45',
   medium: '#D6BF57',
   low: '#54B399',
 } as const;
 
+export const getSeverityColor = (euiTheme: EuiThemeComputed) =>
```

Comment:
I see that @CAWilson94 is doing a similar work [here](https://github.com/elastic/kibana/pull/204007/files#diff-7ffe7d2e7094cc9c2ec6c3d19fc216ba020f6c5b857fcccb28dbfac3f151dfb3R10), so I wonder if there would be a value in extracting this logic into a more generic function/hook that could be moved into our common folder.